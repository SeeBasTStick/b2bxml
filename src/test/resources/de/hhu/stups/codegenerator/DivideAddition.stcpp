int main() {
    Divide divide;
    divide.calculate();
    cout << divide.getRes();
    return 0;
}