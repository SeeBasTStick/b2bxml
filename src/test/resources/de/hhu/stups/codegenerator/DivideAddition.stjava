public static void main(String[] args) {
    Divide divide = new Divide();
    divide.calculate();
    System.out.println(divide.getRes());
}