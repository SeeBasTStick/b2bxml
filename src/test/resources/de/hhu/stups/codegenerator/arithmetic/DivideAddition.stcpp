int main() {
    Divide arithmetic;
    arithmetic.simulate();
    cout << arithmetic.getValue();
    return 0;
}