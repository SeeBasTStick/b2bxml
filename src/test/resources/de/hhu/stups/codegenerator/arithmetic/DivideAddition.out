public static void main(String[] args) {
    Divide arithmetic = new Divide();
    arithmetic.simulate();
    System.out.println(arithmetic.value);
}